static/js/components/SourceTable.jsx

```diff
@@ -347,6 +409,20 @@ const RenderShowClassification = ({ source }) => {
         disableTouchListener
         title={
           <>
+            <Button
+              key={source.id}
+              id="down_vote"
+              onClick={() => addVotes(downvoteValue)}
+            >
+              <ThumbDown color={downvoteColor} />
+            </Button>
+            <Button
+              key={source.id}
+              id="up_vote"
+              onClick={() => addVotes(upvoteValue)}
+            >
+              <ThumbUp color={upvoteColor} />
+            </Button>
```

Comment:
I was experimenting with the thumbs up/down box for all classifications. Might the below box reduce unintentional clicks of the delete-all button?
<img width="200" alt="Screen Shot 2023-01-11 at 10 55 34 AM" src="https://user-images.githubusercontent.com/42810347/211871089-60ea1914-9e27-466a-8f43-c674bd3cca43.png">

Reply:
@bfhealy It's a bit up to you. I expect you will be a power user of this feature, so whatever you think is best, feel free to push it.

Reply:
@mcoughlin Just pushed the new box - I think it will be beneficial to keep the three buttons more separated.